src/SWP/Bundle/CoreBundle/Controller/ContainerController.php

```diff
@@ -14,6 +14,7 @@
 
 namespace SWP\Bundle\CoreBundle\Controller;
```

Comment:
this controller can stay in TemplatesSystemBundle now since we decoupled it from fos rest bundle